doc/diffs.md

```diff
@@ -68,53 +166,184 @@ The "jit-diff list" command has this help message:
+On non-Windows, consult the test instructions
 [here](https://github.com/dotnet/coreclr/blob/master/Documentation/building/unix-test-instructions.md).
 Note that you can pass `--testDir=NONE` to runtest.sh to get the
 same effect as passing `GenerateLayoutOnly` to runtest.cmd on Windows.
 
-## jit-analyze
+This command will generate the diffs into the specified output directory. It will automatically create a "tag"
+subdirectory (if the `--tag` switch isn't specified to override the default name), and within that tag
+directory will be "base" and "diff" directories, containing the diffs. The default tag directory looks like
+`dasmset_12`, with a unique number for every run of jit-diff.
 
-The jit-analyze tool understands the format of the `*.dasm` files in a diff and can extract
-this data to produce a summary and/or dump the info to a data file (in CSV or JSON format).
-The common usage of the tool is to extract interesting diffs, if any, from a diff run as
-part of the development progresses.
+jit-analyze will be run to compare the generated output.
 
-This tool can be invoked automatically after generating diffs using the `-a` option to `jit-diff diff`.
+You can also run a recursive textual comparison tool like windiff on Windows to visually compare
+the diffs, e.g.:
+```
+    c:\gh\coreclr> windiff c:\diffs\dasmset_12\base c:\diffs\dasmset_12\diff
+```
 
-Sample help command line:
+## Simplified jit-diff usage: defaults
+
+As seen above, specifying all required information can be quite verbose. jit-diff can automatically determine
+most of the arguments using computed defaults. The above diff can be accomplished using simply:
+```
+    c:\gh\coreclr> jit-diff diff --base_root e:\gh\coreclr2
+```
+
+The defaults are:
+* If jit-diff is invoked with the current directory within the dotnet/coreclr repo, then the root of
+  this repo serves to find the diff compiler and `CORE_ROOT` directory. (Note that we have no reasonable
+  default for determining what the baseline toolset or repo is. This is specified with the `--base_root` argument.)
+* The default output directory is `<repo_root>\diffs` (in this case, c:\gh\coreclr\diffs).
+* The default diff architecture is x64. If this isn't found, x86 is tried.
+* The default diff build flavor is checked. If this isn't found, debug is tried.
+* By default, diffs are done using System.Private.CoreLib.dll. (That is, `--corelib` is the default.)
+
+To instead do diffs over the framework assemblies (not just System.Private.CoreLib.dll), using an x86
+debug build, run:
+```
+    c:\gh\coreclr> jit-diff diff --frameworks --arch x86 --build debug --base_root e:\gh\coreclr2
+```
+
+To simplify this more, create a dotnet/coreclr repo clone that you will always use for baselines.
+Create a configuration file that specifies your baseline root directory, specifying a `base_root`
+default. See the document [configuring defaults](config.md) for details.
```

Comment:
> I would prefer if the tool automatically found and downloaded an appropriate baseline. As that seems difficult (or impossible)

I'm not following why cijobs couldn't be expanded to make that doable.  Seems like the main things lacking are:
 - cijobs having a way to find a job that matches a given commit hash, instead of having to specify by jenkins run ID
 - jit-diff having a way to figure out what baseline you want (might be as simple as `git merge-base HEAD coreclr/master`)
 - jit-diff knowing which jobs build which compilers

>I find that to be too error prone... I find it easier to just keep a baseline clone around left untouched, and sync'ed at the same time as my diff branch.

Sure, you do you.  I find it too error prone *not* to do it that way.  The main point is that the tool needs to support all our workflows.